packages/tests/server/tupleson.test.ts

```diff
@@ -402,6 +403,65 @@ describe('with transformer', () => {
   });
 });
 
+describe.todo('with tupleson transformer', () => {
+  const orderedResults: number[] = [];
+  const tuplesonOptions: TsonOptions = {
+    nonce: () => '__tson',
+    types: [tsonBigint],
+  };
+  const ctx = konn()
+    .beforeEach(() => {
+      const t = initTRPC.create({
+        transformer: 'tupleson', // <------- help, idk if this is the right thing
```

Comment:
To me, it depends, if `tupleson` is the default anyway, the string is fine, if not I would opt for a reference from the `tupleson` package in order to make it tree shakable

The tree shake will, however, only fork if the default is different from `tupleson`, or if this is a required property and thus none is the default...

Reply:
> To me, it depends, if `tupleson` is the default anyway, the string is fine, if not I would opt for a reference from the `tupleson` package in order to make it tree shakable
> 
> The tree shake will, however, only fork if the default is different from `tupleson`, or if this is a required property and thus none is the default...

@Akxe
We've been leaning towards making Tupleson *the* transformer moving forward, deprecating the need for this field. It would make TSON a hard requirement for future implementations (but we maintain backward compat for a bit).

If users want to customize the transform logic, they do so by configuring TSON (which allows an alternative to the default `JSON` from stringifying/parsing).

Essentially, we would be moving the 'transformer' concept to be an implementation detail of Tupleson, instead of an implementation detail of tRPC.

That was an idea, at least. Thoughts?

Reply:
I do like the idea of using TSON. My point was merely informative :)

Reply:
I do not want a string to be there. It means that even if you were to replace it, the TSON dependency is forced upon the consumer. 

From my point of view; no choice or choice in the form of a `function` or a `Class`.

Why string? What would you put there otherwise?